components/base/wheeled/wheeled_base.go

```diff
@@ -147,7 +150,7 @@ func (base *wheeledBase) runAll(ctx context.Context, leftRPM, leftRotations, rig
 		fs = append(fs, func(ctx context.Context) error { return m.GoFor(ctx, rightRPM, rightRotations, nil) })
 	}
 
-	if _, err := rdkutils.RunInParallel(ctx, fs); err != nil {
+	if _, err := rdkutils.RunInParallel(ctx, fs); err != nil && !errors.Is(err, context.Canceled) {
```

Comment:
For reviewers: this line silences a cancel context error, the underlying `RunInParallel` is also trying to silence that error, but it seems like one of the cancel contexts from the parallel functions is actually coming through.

Reply:
I don't think run in Parallel tries to silence the context Cancelled error but just tries to store it once so I think it make sense for you to check if there was a cancellation.